joomla/joomla-cms
# [4.0] Core.js cleanup Pt1

Pull Request for Issue # .

### Summary of Changes
- The function `Joomla.popupWindow()` moved to toolbar.js where it belongs
- The function `window.writeDynaList()` moved to system\fields\moduleorder.js where it belongs
- The function `window.changeDynaList()` was removed completely as it had no real usage
- The function `window.radioGetCheckedValue()` was removed completely as it had no real usage

- All the code in core.js was just muted (to help the syncing with 3.x)

### Testing Instructions
Go to Administrator-> articles and press the help button on the toolbar, you should see a new window (with no toolbar and an error message, if so then this PR works)


### Expected result



### Actual result



### Documentation Changes Required
Deprecations need to be done in the 3.X branch

@mbabker @wilsonge any objections here?
